provelo_customization/wizard/hr_holidays_summary_department_view.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <menuitem
+            name="Leaves by Department"
+            parent="hr_holidays.menu_hr_holidays_report"
+            action="hr_holidays.action_hr_holidays_summary_dept"
+            id="hr_holidays.menu_account_central_journal"
+            icon="STOCK_PRINT"
+            groups="base.group_hr_manager,base.group_hr_user"/>
```

Comment:
All `base.group_hr_manager` users are also `base.group_hr_user`:

``` 
    <record id="group_hr_manager" model="res.groups"> ...
        <field name="implied_ids" eval="[(4, ref('group_hr_user'))]"/> ...
    </record>
```

Reply:
But `base.group_hr_user` are not also `base.group_hr_manager`... Not sure to understand :p

Reply:
That means, if you want to give it to `group_hr_user` and `group_hr_manager`, giving the rights to `group_hr_user` is enough :-)

Reply:
changed in last commit